src/ContainerAppHelper.ts

```diff
@@ -0,0 +1,438 @@
+import * as core from '@actions/core';
+import * as io from '@actions/io';
+import * as path from 'path';
+import * as os from 'os';
+import { CommandHelper } from './CommandHelper';
+import { Utility } from './Utility';
+import util = require('util');
+const cpExec = util.promisify(require('child_process').exec);
```

Comment:
I don't think we need to use this anymore -- we should be OK to use `@actions/exec`

Reply:
Updated.